test/testvm.py

```diff
@@ -158,7 +188,8 @@ def build(self, args):
 
     def run_setup_script(self, script, args):
         """Prepare a test image further by running some commands in it."""
-        self.start(maintain=True)
+        # run this on the original image, not the one in the run directory
+        self.start(maintain=True, original=True)
```

Comment:
Hmm, this looks wrong.  The original image in $TEST_DATA/images should never be changed, except when it is overwritten completely by vm-save.

All activity should happen with the image in ./run.  When starting a VM with the image in ./run, the changes made from within that VM are either saved back to the image (during vm-create and vm-install) or thrown away (during check-*).